src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SingleLineJavadocCheck.java

```diff
@@ -58,6 +58,45 @@
  * <pre>
  * &lt;module name=&quot;SingleLineJavadoc&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *   public class Test {
+ *     /** &#64;return integer value *&#47; // violation
```

Comment:
Please use `{@literal @}` instead of `&#64;`

Reply:
Sir, putting this tag gives Xdocs error.

Reply:
what kind of error? In xdoc you can put same thing and it will work fine, there are examples of using this in xdocs

Reply:
@ritvij14 ok, please ignore this items than